docs/openapi.yaml

```diff
@@ -234,7 +234,10 @@ paths:
       tags:
         - Transactions
       operationId: get_mempool_transaction_list
-      description: Retrieves all transactions that have been recently broadcast to the mempool. These are pending transactions awaiting confirmation.
+      description: |
+          Retrieves all transactions that have been recently broadcast to the mempool. These are pending transactions awaiting confirmation.
+
+          If you need to actively monitor new transactions, we highly recommend subscribing  to [WebSockets or Socket.io](https://github.com/hirosystems/stacks-blockchain-api/tree/master/client) for real-time updates.
```

Comment:
Small nit: there's double space between the words "subscribing" and "to" in this and the other sentences.